src/main/java/fr/pilato/elasticsearch/crawler/fs/client/ElasticsearchClient.java

```diff
@@ -55,6 +55,7 @@
     private final Elasticsearch settings;
     private final RestClient client;
     private String FIELDS = null;
+    public static final Integer defaultSearchSize = 10;
```

Comment:
That being said, it's only used in tests once so may be just change the test and use `10`?
That said your change:

```java
SearchResponse response = elasticsearchClient.search("fscrawler_test_all_documents", null, fullQuery, ElasticsearchClient.defaultSearchSize, null);
```

Is not really related to this PR, right?